mythril/laser/ethereum/instructions.py

```diff
@@ -1332,15 +1391,16 @@ def callcode_(self, global_state: GlobalState) -> List[GlobalState]:
             return [global_state]
 
         transaction = MessageCallTransaction(
-            global_state.world_state,
-            environment.active_account,
-            environment.address,
-            call_data=call_data,
+            world_state=global_state.world_state,
             gas_price=environment.gasprice,
-            call_value=value,
+            gas_limit=2300,  # G_callstipend
```

Comment:
isn't the stipend a lower bound? Or am I misunderstanding that?

Reply:
Yeah. The gas stipend is actually deducted from the call gas costs for a non-zero value transfer. That doesn't really affect the gas limit. That rather comes from the call parameters. I've added the gas value, which is popped from the stack in the latest changes :+1: